src/grisp_tools_package.erl

```diff
@@ -1,13 +1,19 @@
 -module(grisp_tools_package).
 
 % API
--export([list/1]).
+-export([list_online/1]).
+-export([list_local/1]).
 
 %--- API -----------------------------------------------------------------------
 
-list(#{type := Type, platform := Platform}) ->
+list_online(#{type := Type, platform := Platform}) ->
     parse(Type, list_bucket(Type, [<<"platforms/">>, atom_to_binary(Platform)])).
 
+list_local(otp) ->
```

Comment:
Don't we cache packages per platform? If not, would this list packages for all platforms and how would that affect the deployment?

Reply:
So we could in theory deploy with the wrong package?

Reply:
This needs to be addressed